mono/metadata/object.c

```diff
@@ -6325,8 +6325,10 @@ mono_string_new_checked (MonoDomain *domain, const char *text, MonoError *error)
 	
 	if (!eg_error)
```

Comment:
Great to see this PR in general, so I know where to test lots of string stuff, thank you.